.fuzzbuzz.yml

```diff
@@ -1,4 +1,4 @@
-base: ubuntu:focal
+base: debian:buster-backports
```

Comment:
I've explained in the PR description, it's because `buster` has a glibc version that is at the minimum version that we support, read the description for why we cannot use a higher version and simply not include newer symbols.